account_invoice_change_currency/tests/test_account_invoice_change_currency.py

```diff
@@ -0,0 +1,320 @@
+                    login="fm",
+                    email="accountmanager@yourcompany.com",
+                    groups_id=[(6, 0, [res_users_account_manager.id])],
+                )
+            )
+        )
+
+        # Needed to create invoice
+        self.account_type = self.env["account.account.type"].create(
+            {
+                "name": "acc type test 2",
+                "type": "other",
+                "include_initial_balance": True,
+                "internal_group": "asset",
+            }
+        )
+        self.account_account_line = self.env["account.account"].create(
+            {
+                "name": "acc inv line test",
+                "code": "X2021",
+                "user_type_id": self.account_type.id,
+                "reconcile": True,
+            }
+        )
+        self.product_1 = self.env["product.product"].create({"name": "Product 1"})
+        self.product_2 = self.env["product.product"].create({"name": "Product 2"})
+        self.analytic_account = self.env["account.analytic.account"].create(
+            {"name": "test account"}
+        )
+        self.tax_account = self.env["account.account"].search(
+            [
+                (
+                    "user_type_id",
+                    "=",
+                    self.env.ref("account.data_account_type_current_assets").id,
+                )
+            ],
+            limit=1,
+        )
+
+    def create_simple_invoice(self, date=False, context=None, inv_type=None):
+        if not context:
+            context = {}
+        context["default_move_type"] = True
+        invoice_lines = [
+            (
+                0,
+                0,
+                {
+                    "product_id": self.product_1.id,
+                    "quantity": 5.0,
+                    "price_unit": 142.0,
+                    "name": "Product that cost 142",
+                    "account_id": self.account_account_line.id,
+                },
+            ),
+            (
+                0,
+                0,
+                {
+                    "product_id": self.product_2.id,
+                    "quantity": 4.0,
+                    "price_unit": 213.0,
+                    "name": "Product that cost 213",
+                    "account_id": self.account_account_line.id,
+                },
+            ),
+        ]
+        invoice = (
+            self.env["account.move"]
+            .with_user(self.manager)
+            .with_context(**context)
+            .create(
+                {
+                    "partner_id": 1,
+                    "move_type": inv_type or "in_invoice",
+                    "invoice_date": date,
+                    "currency_id": self.env.ref("base.EUR").id,
+                    "invoice_line_ids": invoice_lines,
+                    "state": "draft",
+                }
+            )
+        )
+        return invoice
+
+    def test_set_original_price_unit(self):
```

Comment:
You don't need a new test for this, an invoice is being created in several tests already